docs/getting-started-guides/windows/index.md

```diff
@@ -1,149 +1,256 @@
 ---
-title: Windows Server Containers
+title: Using Windows Server Containers in Kubernetes
 ---
-**Note:** These instructions were recently updated based on Windows Server platform enhancements
+**Note:** These instructions were recently updated based on Windows Server platform enhancements and the Kubernetes v1.9 release
 
-Kubernetes version 1.5 introduced support for Windows Server Containers based on the Windows Server 2016 OS. With the release of Windows Server version 1709 and using Kubernetes v1.8 users are able to deploy a K8s cluster either on-premises or in a private/public cloud using a number of different network toplogies and CNI plugins. Platform improvements include:
-- Improved support for Pods! Shared network namespace (compartment) with multiple Windows Server containers (shared kernel)
-- Reduced network complexity by using a single network endpoint per Pod
+Kubernetes version 1.5 introduced Alpha support for Windows Server Containers based on the Windows Server 2016 operating system. With the release of Windows Server version 1709 and using Kubernetes v1.9 users are able to deploy a Kubernetes cluster either on-premises or in a private/public cloud using a number of different network topologies and CNI plugins. Some key feature improvements for Windows Server Containers on Kubernetes include:
+- Improved support for pods! Shared network namespace (compartment) with multiple Windows Server containers (shared kernel)
+- Reduced network complexity by using a single network endpoint per pod
 - Kernel-Based load-balancing using the Virtual Filtering Platform (VFP) Hyper-v Switch Extension (analogous to Linux iptables)
+- Container Runtime Interface (CRI) pod and node level statistics
+- Support for kubeadm commands to add Windows Server nodes to a Kubernetes environment
 
- The Kubernetes control plane (API Server, Scheduler, Controller Manager, etc) continue to run on Linux, while the kubelet and kube-proxy can be run on Windows Server version 1709.
+The Kubernetes control plane (API Server, Scheduler, Controller Manager, etc) continue to run on Linux, while the kubelet and kube-proxy can be run on Windows Server 2016 or later
 
-**Note:** Windows Server Containers on Kubernetes is an Alpha feature in Kubernetes 1.8.
+**Note:** Windows Server Containers on Kubernetes is a Beta feature in Kubernetes v1.9
+
+## Build
+We recommend using the release binaries that can be found at [https://github.com/kubernetes/kubernetes/releases](https://github.com/kubernetes/kubernetes/releases).
+
+If you wish to build the code yourself, please follow the next instructions:
```

Comment:
We intend to have these instructions included on our MSDN docs page. Those instructions are currently staged here: https://github.com/gkudra-msft/Virtualization-Documentation/blob/kubernetes-docs/virtualization/windowscontainers/kubernetes/compiling-kubernetes-binaries.md

My intent is to have the contents from this PR show up on K8s doc site - Getting Started Guide - and then have links to our MSDN docs for more detailed information such as how to build the binaries. Any concerns?

Reply:
If you're referring to to https://github.com/Microsoft/SDN/blob/master/Kubernetes/HOWTO-on-prem.md which was before as MSDN docs, I don't think that's really ok in my opinion.
I mentioned some reasons in the original PR: https://github.com/kubernetes/website/pull/6180/files#r153472849
And besides that you are referencing a master branch of a github repository, there should be at least a different branch.
Any reason not to have those few lines on how to build the code here and to have them somewhere else?

Reply:
i prefer to keep a single source of truth on how to build K8s for windows in this page. this is a kubernetes docs page after all and we should not be referencing outside if we can avoid it